scripts/release.ts

```diff
@@ -648,6 +656,30 @@ const npmUpdateVersionForPrerelease = (ctx: TaskContext) => {
     updateFile(`${ctx.packageJSONFilePath}`, `${JSON.stringify(ctx.packageJSONFileContent, null, 2)}\n`);
 };
 
+const releaseScriptCanBeRun = async (): Promise<boolean> => {
+
+    // Check if on `master`.
+
+    if (await gitGetCurrentBranch() !== 'master') {
+        console.log('Release cannot be run as the branch is not `master`.');
```

Comment:
Nit: `console.error`

Also, should we check that it's pointing to webhintio/hint?  I could have a clone of my fork, be in the master branch and still try to publish something that shouldn't. Not that it has ever happened to me before 🙄

Reply:
@webhintio/contributors Can you all do `git branch -vv` and print the output? Thanks!

Reply:
![image](https://user-images.githubusercontent.com/1581288/51053490-ac31e080-158e-11e9-8148-d0808aa0ad01.png)

Reply:
On Windows (output is the same with WSL)
![image](https://user-images.githubusercontent.com/606594/51053637-092d9680-158f-11e9-9037-6b31765b06a1.png)

Reply:
On Windows:
```
  browser-stress-tests                   06283038 Chore: Hooks into the devtools
  compat-configuration                   1ddab1b5 New: Enable compat-api hints by default
  extension-browser-firefox-repro        262f55c9 Chore: Strip down repro
  extension-vscode-e2e-tests             a840c637 Chore: Add a test launch target
  greenkeeper/default/@types/sinon-5.0.7 ecbb9784 [upstream/greenkeeper/default/@types/sinon-5.0.7: gone] Chore: Update `@types/sinon` to `v5.0.7`
* master                                 072fe57c [origin/master] Chore: Update `vscode` `devDependency` and `engine` to latest versions
```

Reply:
Ok, thanks.

Ready for another review.